src/shared/codelens/codeLensUtils.ts

```diff
@@ -64,7 +64,10 @@ export async function makeCodeLenses({ document, token, handlers, language }: {
             language
         }
         lenses.push(makeLocalInvokeCodeLens({ ...baseParams, isDebug: false }))
-        lenses.push(makeLocalInvokeCodeLens({ ...baseParams, isDebug: true }))
+        if (language !== 'python') {
```

Comment:
Can we also show Debug codelenses when language is python and os is win32?

Reply:
✔️